datahub-actions/src/datahub_actions/entrypoints.py

```diff
@@ -30,11 +31,25 @@
 BASE_LOGGING_FORMAT = (
     "[%(asctime)s] %(levelname)-8s {%(name)s:%(lineno)d} - %(message)s"
 )
+LOGGING_FILE_LOCATION = "/tmp/datahub/logs/actions/actions.out"
 logging.basicConfig(format=BASE_LOGGING_FORMAT)
 
 MAX_CONTENT_WIDTH = 120
 
 
+class LogFilter(logging.Filter):
+    """Filters (lets through) all messages with level < LEVEL"""
+
+    # https://stackoverflow.com/questions/1383254/logging-streamhandler-and-standard-streams
+    def __init__(self, level):
+        self.level = level
+
+    def filter(self, record):
+        # "<" instead of "<=": since logger.setLevel is inclusive, this should
+        # be exclusive
+        return record.levelno < self.level
```

Comment:
It is more intuitive if you pass in the same level as the one you pass to `setLevel` on the handler.
Then you can use `<=` here.